mappings/net/minecraft/screen/ScreenHandler.mapping

```diff
@@ -44,6 +49,31 @@ CLASS net/minecraft/class_1703 net/minecraft/screen/ScreenHandler
 		ARG 2 clickData
 		ARG 3 actionType
 		ARG 4 player
+	METHOD method_34245 setPreviousTrackedSlot (ILnet/minecraft/class_1799;)V
+		ARG 1 slot
+		ARG 2 stack
+	METHOD method_34246 updateTrackedSlot (ILnet/minecraft/class_1799;Ljava/util/function/Supplier;)V
+		ARG 1 slot
+		ARG 2 stack
+		ARG 3 copySupplier
+	METHOD method_34247 copySharedSlots (Lnet/minecraft/class_1703;)V
+		ARG 1 handler
+	METHOD method_34248 updateSyncHandler (Lnet/minecraft/class_5916;)V
+		ARG 1 handler
+	METHOD method_34250 setPreviousCursorStack (Lnet/minecraft/class_1799;)V
+		ARG 1 stack
+	METHOD method_34252 syncState ()V
+	METHOD method_34253 updateSlot (ILnet/minecraft/class_1799;Ljava/util/function/Supplier;)V
+		ARG 1 slot
+		ARG 2 stack
+		ARG 3 copySupplier
+	METHOD method_34254 setCursorStack (Lnet/minecraft/class_1799;)V
+		ARG 1 stack
+	METHOD method_34255 getCursorStack ()Lnet/minecraft/class_1799;
+	METHOD method_34256 disableSyncing ()V
+	METHOD method_34257 enableSyncing ()V
+	METHOD method_34258 updateCursorStack ()V
+	METHOD method_34259 getCursorCommandItemSlot ()Lnet/minecraft/class_5630;
```

Comment:
```suggestion
	METHOD method_34259 getCursorSlot ()Lnet/minecraft/class_5630;
```

Reply:
I think this name is misleading, since `CommandItemSlot`s (or whatever we end up naming them) aren't actually related to normal slots